sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -1043,7 +1043,7 @@ public virtual Response UpdateEntity<T>(T entity, ETag ifMatch, TableUpdateMode
         /// For example, the following expression would filter entities with a PartitionKey of 'foo': <c>e => e.PartitionKey == "foo"</c>.
         /// </param>
         /// <param name="maxPerPage">
-        /// The maximum number of entities that will be returned per page.
+        /// The maximum number of entities that will be returned per page. The default value, if unspecified, is 1000 for storage accounts and is not limited for Cosmos DB table API.
```

Comment:
```suggestion
        /// The maximum number of entities that will be returned per page. If unspecified, the default used is 1000 for storage accounts and is not limited for Cosmos DB table API.
```

The current version reads a bit awkward to me.  The default value is still the default whether overridden by configuration or not.   Maybe something like the above?